### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [BankoP/TeaCozy#1]
Images should have descriptive alt text. Screen readers use the alt text to read out loud for the visually impaired. This tells them what is on the screen.
